_posts/2023-06-05-timelines-save-time-and-money.md

```diff
@@ -0,0 +1,294 @@
+---
+layout: post
+title: "It’s Time for a Time-centric Stack: Timeline Abstractions Save Time and Money"
+image: assets/images/blog/Blog_Post_Time_Centric.png
+author: Brian Godsey
+---
+
+
+
+When processing and building real-time feature values from event-based data, it
+is easy to underestimate the power of using tools that are built to understand
+how time works. Many software tools can parse and manipulate time-based data,
+but the vast majority have little more than convenience functionality for
+parsing and doing arithmetic on date-time values. On the other hand, Kaskada has
```

Comment:
This sentence `On ...` feels like a run-on. I think it can be split at the "just works by default. This understanding is very powerful: it reduces development time and effort, simplifies deployment and debugging, and reduces the compute and storage resources of real-time event-based systems".

It also clearly connects the benefits to the "power".